lms/templates/learner_dashboard/course_card_actions.underscore

```diff
@@ -0,0 +1,11 @@
+<% if(enrollment_status){ %>
+    <div class="enrollment-info"><%- gettext('ENROLLED') %></div>
+    <a id="view_course_link" href="<%- courseware_url %>" class="btn-neutral btn view-course-link">
+        <%- gettext('View Course') %>
+    </a>
+<% }else{ %>
+    <div class="enrollment-info"><%- gettext('NOT ENROLLED') %></div>
```

Comment:
I think it is more flexible to have the text lowercase and use CSS to display it all uppercase.